tests/functional/test_matrix_service.py

```diff
@@ -1,16 +1,15 @@
-import unittest
+import json
 import os
 import requests
-import json
 import time
+import unittest
```

Comment:
For consistency:
_built-in packages_
\n
_installed packages_
\n
_local imports_

Reply:
So you're saying `requests` is out of order here?

Reply:
Yea requests should be in the 2nd block and the other 4 in the first